model/work.py

```diff
@@ -218,7 +218,7 @@ class Work(Base):
     # A precalculated MARC record containing metadata about this
     # work that would be relevant to display in a library's public
     # catalog.
-    marc_record = Column(Binary, default=None)
+    marc_record = Column(String, default=None)
```

Comment:
I did a preliminary test of this code when converted to Python 3 and it seems to work -- data is read from a `String` field as a bytestring. A `Unicode` field is stored as a bytestring in the database, but decoded as UTF-8 on the way into Python code.